frontend/src/app/supportAdmin/ManageUsers/operationMocks.ts

```diff
@@ -0,0 +1,151 @@
+import {
```

Comment:
Oh! I like how you separated this out so we don't have one massive file. Will do that in my future PRs as well. 👀